File: AzW.Web.Portal/src/models/VNet.js
import ResourceType from './ResourceType'
export default class VNet
{
    constructor (){
        this.GraphModel = {
            Id: '',
            ResourceType: ResourceType.VNet(),
            DisplayName: ''
        };

        this.ProvisionContext = {
            ResourceType: ResourceType.VNet(),
           
            Name: '',
            Location: '',
            ResourceGroupName: '',
            AddressSpace: '',
            Subnets: []
        }; 
        this.CalculatorContext = {
            IsFree: true,
            Tier: ''
        }
    }
}

File: AzW.Model/Provision/VNet.cs

using System.Collections.Generic;

namespace AzW.Model
 {
     public class VNet
     {
          public string Name { get; set; }
          public string Location { get; set; }
          public string ResourceGroupName { get; set; }
          public string AddressSpace { get; set; }
         public List<Subnet> Subnets { get; set; }
     }
 }